lib/matplotlib/tight_layout.py

```diff
@@ -81,23 +60,19 @@ def auto_adjust_subplotpars(
         margin_left = margin_bottom = margin_right = margin_top = None
     else:
         margin_left, margin_bottom, _right, _top = rect
-        if _right:
-            margin_right = 1 - _right
-        else:
-            margin_right = None
-        if _top:
-            margin_top = 1 - _top
-        else:
-            margin_top = None
+        margin_right = 1 - _right if _right else None
+        margin_top = 1 - _top if _top else None
 
-    vspaces = [[] for _ in range((rows + 1) * cols)]
-    hspaces = [[] for _ in range(rows * (cols + 1))]
-
-    union = Bbox.union
```

Comment:
Was this an optimization?

Reply:
I think it was just a typing optimization :); saving an attribute access is unlikely to matter here especially given that we're about to rerender the figure after tightlayouting it...